addons/60fps/addon.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "60FPS project player mode",
-  "description": "Alt+Click the green flag to toggle 60FPS.",
+  "name": "Higher project framerate mode",
+  "description": "Lets you run projects at more frames per second (FPS) so they look smoother. Hold Alt and click the green flag to toggle between Scratch's default 30 FPS and a custom framerate set below.",
   "info": [
     {
-      "type": "notice",
-      "text": "Most projects will not behave properly when running at 60FPS, because increasing the frame rate also increases the speed at which scripts run. This should only be used on projects that support 60FPS.",
+      "type": "warning",
```

Comment:
This might have been brought up already but does this need to be a warning? I don't think it does since it's still not the default when loading a project.

Reply:
Yes, but notices tell the user in which situations the addon might not work, and warnings say how the addon may hinder the user experience.

Reply:
> Yes, but notices tell the user in which situations the addon might not work, and warnings say how the addon may hinder the user experience.

Not necessarily. The following are all notices:
* `cat-blocks`: The "watch mouse cursor" setting may impact performance when the editor is open.
* `better-img-uploads`: Avoid using the HD upload button if you plan to edit the image after uploading.
* `turbowarp-player`: "Replace player" mode changes the Scratch player to a TurboWarp embed, which doesn't support seeing inside projects.

Reply:
I think a warning is appropriate if simply enabling the addon has negative effects. In this case, enabling the addon isn't enough to make projects behave incorrectly - 60 FPS mode needs to be activated.